plugins/sigma.renderers.linkurious/canvas/sigma.canvas.nodes.def.js

```diff
@@ -157,9 +169,9 @@
       // Border:
       if (borderSize > 0) {
         context.beginPath();
-        context.fillStyle = settings('nodeBorderColor') === 'node' ?
-          (color || defaultNodeColor) :
-          settings('defaultNodeBorderColor');
+        context.fillStyle = settings('nodeBorderColor') === 'node'
+          ? borderColor
```

Comment:
if `settings('nodeBorderColor') === 'node'` then the border must be of the color of the node, please use another test value to assign `borderColor`